Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM golang:1.16 as build
 
 COPY . /usr/src/rds_exporter
 
-RUN cd /usr/src/rds_exporter && make build
```

Comment:
hm, this make Docker file useless. could you please open new issue to refactor Makefile and docker file, because this one now make no sense as it would have no binaries.